llvm/test/Verifier/preallocated-invalid.ll

```diff
@@ -65,7 +65,7 @@ define void @preallocated_one_call() {
     ret void
 }
 
-; CHECK: must be a constant
+; CHECK: immarg operand has non-immediate parameter
```

Comment:
Missing test for the call_preallocated_arg case?

Reply:
Fair, but that's a pre-existing test coverage gap. This stuff relies on shared infrastructure. How much testing of immarg do we need, especially for a feature which was never productionized?